### PR TITLE
Add helper struct `ZelbetWitnesses`

### LINKED ISSUE #21: Make zelbet_variable a struct
**Describe what you want implemented** 
Create a struct from the function `zelbet_variables`

**Describe "Why" this is needed**
We use the output of that function in various places but it is never clear what fields we are accessing. Creating a struct that would be initialized once and can then be passed around would fix this issue.

**Describe alternatives you've considered**
NA

**Additional context**
NA

### PR DESCRIPTION
**Pull request by moCello**

### CODE REVIEW

File: src/utils/zelbet_witnesses.rs
Comment (vlopes11) [inline src/utils/zelbet_witnesses.rs:12]: If we are encapsulating the zelbet witness logic, then we should create an API that clarifies its usage. By making all the fields public, we don't improve its logical description

Comment (moCello) [reply src/utils/zelbet_witnesses.rs:12]: With more recent changes to the repo, I realized that we might not even need the struct `ZelbetWitnesses` after all.
Since the struct is actually only used in `bar`, we can also think of creating the relevant `Witnesses` only there.
I created and issue for it #36

Comment (ZER0) [reply src/utils/zelbet_witnesses.rs:12]: @moCello way better if we can remove this struct. Can we close this PR then?

Comment (moCello) [reply src/utils/zelbet_witnesses.rs:12]: @ZER0 I will have to merge this PR first because other PRs are based on this one. I will remove the struct in #36 after merging open PRs. Can I have your approval for this one?

**Review by vlopes11 — APPROVED** [review]
LGTM

**Pull request merged**
